src/backend/distributed/citus--7.0-2--7.0-3.sql

```diff
@@ -0,0 +1,86 @@
+/* citus--7.0-2--7.0-3.sql */
+
+ALTER SEQUENCE pg_catalog.pg_dist_shard_placement_placementid_seq
+  RENAME TO pg_dist_placement_placementid_seq;
+
+ALTER TABLE pg_catalog.pg_dist_shard_placement
+  ALTER COLUMN placementid SET DEFAULT nextval('pg_catalog.pg_dist_placement_placementid_seq');
+
+CREATE TABLE citus.pg_dist_placement (
+  placementid BIGINT NOT NULL default nextval('pg_dist_placement_placementid_seq'::regclass),
+  shardid BIGINT NOT NULL,
+  shardstate INT NOT NULL,
+  shardlength BIGINT NOT NULL,
```

Comment:
And [we assume](https://github.com/citusdata/citus/blob/master/src/backend/distributed/master/master_metadata_utility.c?utf8=%E2%9C%93#L741) `NOT NULL`